x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.scss

```
@@ -0,0 +1,21 @@
+.documents-search-experience {
+  .sui-results-container {
+    flex-grow: 1;
+    padding: 0;
+  }
+
+  .documents-search-experience__sidebar {
+    flex-grow: 1;
+    min-width: 300px;
```

Not 100% sure, but I think we may still want to use `$euiSize` for this?... I think it's an accessibility thing (e.g., for layouts that override $euiSize's default for smaller or larger sizes - a static px doesn't scale accordingly)

```suggestion
    min-width: $euiSize * 19;
```

above is 304px which is probably close enough to 300px 🤷

---

Ohhhh interesting. I think that makes sense.